src/commands/update_all.js

```diff
@@ -0,0 +1,29 @@
+const { SlashCommandBuilder, PermissionFlagsBits } = require("discord.js");
+const db = require("../db");
+const { updateUserFromSheet } = require("./helpers");
+
+async function execute(interaction) {
+  const users = db.collection("users").find();
+  const ids_in_guild = (await interaction.guild.members.fetch()).map(
+    (member) => member.user.id
+  );
+
+  users.forEach(async (doc) => {
+    if (doc && ids_in_guild.includes(doc.user_id)) {
+      updateUserFromSheet(doc.email, doc.user_id, interaction.client);
```

Comment:
we should wrap this in try/catch or at least handle the status that's returned from `updateUserFromSheet` to return an error to the user.

Currently, if we get some issue in that function it will not return anything to the user.